src/components/Modeler.vue

```diff
@@ -54,6 +54,7 @@
       @unsetPools="unsetPools"
       @setPools="setPools"
```

Comment:
Not related to this PR, but the naming for our events are inconsistent. The Vue documentation recommends using kebab-case.

Reply:
I've also noticed this...

Reply:
For reference, it's mentioned the docs here: https://vuejs.org/v2/guide/components-custom-events.html. In our case, it definitely doesn't break anything and can easily be missed, but consistency is something I always advocate for.

Ideally, there's an ESLint rule to enforce this kind of stuff, but I didn't find one :( might look into creating one later.

Reply:
The above references have been fixed in this PR.